src/App.js

```diff
@@ -1,7 +1,65 @@
 import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
 import './App.css';
 import Calculator from './components/Calculator';
 
+function NavBar() {
+  return (
+    <nav>
+      <h1>
+        Math Magicians
+        <span> </span>
+        <img alt="magicHat" src="https://img.icons8.com/external-soft-fill-juicy-fish/60/000000/external-wizard-video-game-elements-soft-fill-soft-fill-juicy-fish.png" />
+      </h1>
+      <ul className="navLink">
+        <li className="navItem"><Link to="/"><span>Home</span></Link></li>
+        <span>|</span>
+        <li className="navItem"><Link to="/calculator">Calculator</Link></li>
+        <span>|</span>
+        <li className="navItem"><Link to="/quote">Quote</Link></li>
+      </ul>
+    </nav>
+  );
+}
+
+function Home() {
```

Comment:
- [ ] Please extract this into a single component to make your app maintainable, you can also reuse this component somewhere else and it is always best practice to use an arrow function